crates/api_common/src/community.rs

```diff
@@ -16,6 +16,64 @@ use serde_with::skip_serializing_none;
 #[cfg(feature = "full")]
 use ts_rs::TS;
 
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Create a tag for a community.
+pub struct CreateCommunityTag {
+  pub community_id: CommunityId,
+  pub id_slug: String,
+  pub name: String,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct CommunityTagResponse {
+  pub id: TagId,
+  pub ap_id: DbUrl,
+  pub name: String,
+  pub community_id: CommunityId,
+}
```

Comment:
The responses should really only return views. So tags should either be added to `CommunityView`, like they already are for `PostView` and `CommentView`, or this response should return `{community: Community, tags: Vec<Tag>`}

Reply:
Makes sense to include it in the CommunityView, you dont want to load a separate endpoint for this data. Anyway there shouldnt be a lot of tags in each community.